test/functional/feature_proxy.py

```diff
@@ -41,12 +43,16 @@
 from test_framework.netutil import test_ipv6_local
 
 RANGE_BEGIN = PORT_MIN + 2 * PORT_RANGE  # Start after p2p and rpc ports
-# From GetNetworkName() in netbase.cpp:
-NET_UNROUTABLE = ""
+
+# Networks returned by RPC getpeerinfo, defined in src/netbase.cpp::GetNetworkName()
+NET_UNROUTABLE = "unroutable"
```

Comment:
This is not used?

Reply:
It's used below, line 171: `self.network_test(node, addr, network=NET_UNROUTABLE)`